### ISSUE TITLE
Multiple word-words

### ISSUE THREAD

**Issue by lopsided** [Dragnipur/tiny-angular-wordcloud#9]
I found that when my tags contained multiple words (ie, "World Cup") then I kept getting overlapping words. I tracked this down to the js `span.offsetWidth` which doesn't give the correct value. The way to fix this is to add this to the css:

```
.tangcloud span {
    white-space: pre;
}
```

See plunkr: http://plnkr.co/edit/B8iS3aQ3HSIJfoNEiZvz?p=preview

Happy to submit a PR if you want to add this to the readme and/or example?

**Comment by Dragnipur** [comment]
Good catch! If you'll put in a PR to add this to the readme I'll approve it.
